### ISSUE TITLE
Audit Improvement

### ISSUE THREAD

**Issue by aashutoshrathi** [saadpasta/developerFolio#195]
## Summary
We have served images in next-gen-format, but there are widgets which are serving JPG and also tons of performance, best practices, and accessibility fixes that needed to be made

### Motivation

![image](https://user-images.githubusercontent.com/21199234/85917858-67c9fb80-b87b-11ea-8307-4bbe00658734.png)

Full Report: [Here](https://lighthouse-dot-webdotdevsite.appspot.com//lh/html?url=https%3A%2F%2Fdeveloperfolio.js.org)

**PS:** For images by Twitter the links look like `https://pbs.twimg.com/media/Ea53nYhUwAEUiEx?format=jpg&name=small` if we could somehow intercept the request and change format to webp, it servers webp.
For eg here: `https://pbs.twimg.com/media/Ea53nYhUwAEUiEx?format=webp&name=small`

**Comment by naveen521kk** [comment]
I just see that it is not possible until we actual get tweets and embed each one. I just checked that it uses Twitter's js lib which doesn't support it. Maybe just checking it is better.

**Comment by aashutoshrathi** [comment]
@naveen521kk I think even if leave these images, there are tons of other stuff to be done.

**Comment by naveen521kk** [comment]
If you can maybe list like below one.

- [ ] Like this

This would be helpful to know what to be done.

**Comment by kartik918** [comment]
This was discussed before too, check #82 for more context.

**Comment by WarenGonzaga** [comment]
hi @kartikcho how about creating a new issue for Performance Update? I love to work on it... I have skills in optimization.

**Comment by kartikcho** [comment]
Hi again, I would suggest waiting for this because our PR backlog has built up a bit. The plan is to get all the new features in and then stabilize the codebase a bit after with tweaks and bug fixes here and there. 
I think this issue resolved at that time would be optimal. Hope you'd still be interested then to help in optimizations!

**Comment by WarenGonzaga** [comment]
@kartikcho yes... I'll be using this for my Github page... and I want my github page load fast as possible and SEO friendly. Just assign it to me once the task is ready. Also, if you don't mind me asking I'm a core developer of animate.css and it is good to have some animation in developerFolio?

**Comment by kartikcho** [comment]
Ah, that's great to hear! I hope you like all the new stuff we are adding into this soon.

I like your suggestion and I've been working on adding a bit of __interactivity__ in the experience too. My work on #260 will add animated illustrations and we are already using `react-reveal` for on scroll effects. If you could propose anything that could make things more interesting or a new interactive component design, I'm up for ideas!
